python/src/opendp/extras/examples/__init__.py

```diff
@@ -12,7 +12,7 @@
 from pathlib import Path
 
 
-def _http_get(url: str) -> bytes:
+def _http_get(url: str) -> bytes: # pragma: no cover
```

Comment:
Here and below, if there was presumably coverage before, why add `pragma: no cover` now?

Reply:
When I run the coverage tests locally, the files are already downloaded, resulting in these code paths giving false-positives. I technically only conditionally want to enable pragma no cover when the files exist, or something like that. 🤷

Reply:
Thanks for the reminder on this. I'll file an issue to either add a low-level test of these, or just add an explanation after the `no cover`, so a dev in the future doesn't ask the same question.